webapi/src/test/scala/org/knora/webapi/responders/v1/ResourcesResponderV1Spec.scala

```diff
@@ -905,7 +890,9 @@ class ResourcesResponderV1Spec extends CoreSpec(ResourcesResponderV1Spec.config)
             actorUnderTest ! ResourceContextGetRequestV1(iri = "http://data.knora.org/c5058f3a", resinfo = true, userProfile = SharedAdminTestData.incunabulaProjectAdminUser)
 
             expectMsgPF(timeout) {
-                case response: ResourceContextResponseV1 => compareResourceCompoundContextResponses(received = response, expected = ResourcesResponderV1SpecContextData.expectedBookResourceContextResponse)
+                case response: ResourceContextResponseV1 =>
+                    val responseAsJson = response.toJsValue
+                    assert(responseAsJson == ResourcesResponderV1SpecContextData.expectedBookResourceContextResponse, "book context response did not match")
```

Comment:
If I understand correctly, the correct ordering of the dependent pages (according to their `seqnum`) is part of the query. So a 1:1 comparison is good.

I am just mentioning this because there are some tests in which the results have to be ordered according to the same pattern before they are compared (the ordering does not matter for the correctness of the results).